http-filter-example/http_filter.cc

```diff
@@ -2,7 +2,7 @@
 
 #include "http_filter.h"
 
-#include "server/config/network/http_connection_manager.h"
+#include "extensions/filters/network/http_connection_manager/config.h"
```

Comment:
Filters should no longer have to include this header directly. This is kind of legacy. Do you mind updating the bazel deps and the includes? E.g.,:

https://github.com/envoyproxy/envoy/blob/master/source/extensions/filters/http/buffer/BUILD#L34
https://github.com/envoyproxy/envoy/blob/master/source/extensions/filters/http/buffer/config.h#L4

Thank you!